index.html

```diff
@@ -16,7 +16,7 @@
 </head>
 <body>
     <nav class="navbar navbar-expand-lg navbar-light bg-light fixed-top">
-        <a class="navbar-brand" href="#"><img height="45px" width="140px" src="./static/images/NCI-LOGO-FA-Color-Blue.png"></a> 
+        <a class="navbar-brand" href="#"><img height="45px" width="140px" alt="Connect blue logo" src="./static/images/NCI-LOGO-FA-Color-Blue.png"></a> 
```

Comment:
was this just preference at add an "ALT" tag to this? I can't see where we are referencing this

Reply:
It's not referred. Adding it is to follow the rules about `img` element.

Reply:
oh, is that just an industry standard?